frontend/src/pages/MaengelForm.js

```diff
@@ -56,40 +56,40 @@ export default function MaengelForm({ initialMode, title }) {
   const initializeMangel = () => {
     getUserAddress(token, user.username)
       .then(fetchedProfile => {
-        setProfile([...fetchedProfile])
+        setAddresses([...fetchedProfile])
         setMangel({
           ...initialMangelStates,
           dateNoticed: new Date().getTime(),
           contactLoggerList: [],
-          address: fetchedProfile[1],
+          address: fetchedProfile[0],
         })
       })
       .catch(setError)
-      .finally(() => setLoading(false))
+      .finally(() => {
+        setLoading(false)
+        resetContactLogger()
+      })
   }
 
   const getProfile = () => {
     setLoading(true)
     return getUserAddress(token, user.username)
       .then(fetchedProfile => {
-        setProfile([...fetchedProfile])
+        setAddresses([...fetchedProfile])
       })
       .catch(setError)
   }
 
   useEffect(() => {
-    console.log(mode)
     setLoading(true)
     if (mode === 'new') {
       setReadOnly(false)
       initializeMangel()
-      resetContactLogger()
     } else if (mode === 'view') {
       setReadOnly(true)
       getProfile().then()
```

Comment:
hier musst du nochmal catchen. verstehe auch nicht warum du nichts mit dem then machst

Reply:
ich werde noch mal in einem neuen feature meinen frontende-code überarbeiten